RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -1429,6 +1434,78 @@ boolean auto_knapsackAutoConsume(string type, boolean simulate)
 	return true;
 }
 
+boolean auto_chewAdventures()
+{
+	//tries to chew a size 4 familiar spleen item for adventures
+
+	//these items are all out of standard except in Quantum Terrarium
+	if(!is_unrestricted($item[Unconscious Collective Dream Jar]) && !is_unrestricted($item[Powdered Gold]) && !is_unrestricted($item[beastly paste]))
+	{
+		//powdered gold is also restricted in G lover, but if all these are restricted this must be standard
```

Comment:
powdered **g**old is _not_ restricted in G-Lover (neither is a**g**ua de vida)

also should be using `auto_is_valid` here not `is_unrestricted` as `is_unrestricted` doesn't account for BHY/G-Lover restrictions.

Reply:
this is just to stop more efficiently if everything is blocked by standard. auto_is_valid is already done in chew

Reply:
In that case I think what you want is

```
if(my_path() != "Standard")
	{
		return false;	//all spleen giving adventures are out of standard
	}
```
it won't be future proof for the next in-season path...
but actually I also think it is a mistake entirely. There are no server hits here and checking each item in the list for validity (which the code already does) is fine.

Especially because I think the items are all going to be valid again. When kol re-introduces old items via new method it tends to reset their year for the purpose of standard instead of simply writing up a special exemption for just that one event.
So in this case, I expect the pastes and the like to be back in standard until 2024-01-01.
I can test it tonight by doing an actual standard run

Reply:
they certainly won't be in standard. but since not needed for efficiency check removed  https://github.com/Loathing-Associates-Scripting-Society/autoscend/pull/852